docs/platforms/php/guides/laravel/index.mdx

```diff
@@ -17,17 +17,28 @@ Install the `sentry/sentry-laravel` package:
 composer require sentry/sentry-laravel
 ```
 
-Enable capturing unhandled exception to report to Sentry by making the following change to your `App/Exceptions/Handler.php`:
+Enable capturing unhandled exception to report to Sentry by making the following change to your `bootstrap/app.php`:
```

Comment:
Should we call out that this is now a different file? Do users need to do anything with `App/Exceptions/Handler.php` if they previously had it?

Reply:
This is not a change in the SDK but instead in the framework. But the old way will still work for people upgrading from Laravel 10 to Laravel 11.